lib/httpSignatureStrategy.js

```diff
@@ -0,0 +1,86 @@
+/*!
+ * Copyright (c) 2013-2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const bedrock = require('bedrock');
+const brIdentity = require('bedrock-identity');
+const brKey = require('bedrock-key');
+const Middleware = require('http-signature-middleware');
+let jsigs = require('jsonld-signatures');
+const {promisify} = require('util');
+const BedrockError = bedrock.util.BedrockError;
+let didio = require('did-io');
+let jsonld = require('jsonld');
+
+const api = {};
+module.exports = api;
+
+api.strategy = new Middleware({name: 'signature'});
+api.strategy.use('getKey', _getKey);
+api.strategy.use('getUser', _getUser);
+api.strategy.use('validateRequest', _validateRequest);
+
+bedrock.events.on('bedrock.init', () => {
+  // get configured lib instances
+  jsonld = jsonld();
+  didio = didio();
+  didio.use('jsonld', jsonld);
+  jsonld.documentLoader = didio.createDocumentLoader({
+    baseUrl: bedrock.config.passport.strategies.did.didio.baseUrl,
+    wrap: (url, callback) => bedrock.jsonld.documentLoader(url, callback),
+  });
```

Comment:
I guess this stuff is fine for now to go into `master` ... but we will need to rev it almost right away. I don't think there's a good reason to release a new version of bedrock-passport using the old did-io unless it's totally non-breaking and helps us make progress somewhere else. I think we should be getting pretty close to getting the new did-io.